src/v/http/client.cc

```diff
@@ -75,34 +75,36 @@ void client::check() const {
 
 ss::future<client::request_response_t> client::make_request(
   client::request_header&& header, ss::lowres_clock::duration timeout) {
-    vlog(http_log.trace, "client.make_request {}", header);
     auto verb = header.method();
     auto target = header.target();
+    ss::sstring target_str(target.data(), target.size());
     auto req = ss::make_shared<request_stream>(this, std::move(header));
-    auto res = ss::make_shared<response_stream>(this, verb);
+    auto res = ss::make_shared<response_stream>(this, verb, target_str);
+    prefix_logger ctxlog(http_log, ssx::sformat("{{}}", target_str));
```

Comment:
isn't `ssx::sformat("{{}}", target_str)` just `{}` for any `target_str`? Why can't we just pass the string directly?

Reply:
this is a bug, fixed